### PR TITLE
Feature/change hero basic

### PR DESCRIPTION
**Pull request by alexchal**

### CODE REVIEW

File: src/js/components/molecules/cta-block/cta-block.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            return elem instanceof Button;
+        })
+    }
+
+    get isContentLink ()  {
+        return this.ctas.filter(elem => {
+            return elem instanceof ContentLink;
+        })
+    }
+}
```
Comment (rhelling) [inline src/js/components/molecules/cta-block/cta-block.ts]: Its better to name these buttons and contentLinks so its more clear it is a collection of items.

File: src/js/components/atoms/content-link-list/content-link-list.models.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import { IContentLink } from 'components/atoms/content-link/content-link.models';
+// import { IContentLink } from 'components/atoms/content-link/content-link.models';
```
Comment (zwaardje) [inline src/js/components/atoms/content-link-list/content-link-list.models.ts]: remove comment

File: src/js/components/atoms/content-link-list/content-link-list.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import { Vue, Component, Prop } from 'vue-property-decorator';
-import { IContentLink } from 'components/atoms/content-link/content-link.models';
+// import { IContentLink } from 'components/atoms/content-link/content-link.models';
```
Comment (zwaardje) [inline src/js/components/atoms/content-link-list/content-link-list.ts]: Remove comment

File: src/js/components/atoms/index.js [hunk 4]
```diff
@@ -1,7 +1,7 @@
 export {default as AnimatedArrow} from 'atoms/animated-arrow';
 export {default as AppLink} from 'atoms/app-link';
 export {default as ContentLink} from 'atoms/content-link';
-export {default as ContentLinkList} from 'atoms/content-link-list';
+// export {default as ContentLinkList} from 'atoms/content-link-list';
```
Comment (zwaardje) [inline src/js/components/atoms/index.js]: Remove comment

File: src/js/components/molecules/card/card.models.ts [hunk 5]
```diff
@@ -1,13 +1,13 @@
 import { ITags } from '../tag-list/tag-list.models';
 import { IImage } from '../../atoms/responsive-picture/responsive-picture.models';
 import { IIcon } from '../../atoms/icon/icon.models';
-import { IContentLink } from '../../atoms/content-link/content-link.models';
+// import { IContentLink } from '../../atoms/content-link/content-link.models';
```
Comment (zwaardje) [inline src/js/components/molecules/card/card.models.ts]: Remove comment

File: src/js/components/atoms/content-link-list/content-link-list.models.ts (hunk 2 above)
Comment (croyle) [inline src/js/components/atoms/content-link-list/content-link-list.models.ts]: I would rather remove a line than comment it.
if you want to go back in time we always have git to do so.

File: src/js/components/atoms/content-link-list/content-link-list.ts (hunk 3 above)
Comment (croyle) [inline src/js/components/atoms/content-link-list/content-link-list.ts]: see comment above

File: src/js/components/molecules/index.js [hunk 6]
```diff
@@ -1,6 +1,6 @@
 export {default as AutoComplete} from 'molecules/auto-complete';
 export {default as BrandCard} from 'molecules/brand-card';
-export {default as Card} from 'molecules/card';
+// export {default as Card} from 'molecules/card';
```
Comment (zwaardje) [inline src/js/components/molecules/index.js]: Remove this

File: src/js/components/atoms/index.js (hunk 4 above)
Comment (croyle) [inline src/js/components/atoms/index.js]: see above :)

File: src/js/components/molecules/card/card.models.ts (hunk 5 above)
Comment (croyle) [inline src/js/components/molecules/card/card.models.ts]: see above :)

File: src/js/components/molecules/index.js (hunk 6 above)
Comment (croyle) [inline src/js/components/molecules/index.js]: see above...

**Pull request merged**